### PR TITLE
Add GTK/OpenGL example

### PR DESCRIPTION
**Pull request by trin94**
This is the GTK equivalent to @cosven's Qt/OpenGL example.

Direct link to gist: https://gist.github.com/trin94/a930f2a13cee1c9fd21cab0393bf4663

Also see #99

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -193,7 +193,7 @@ PyQT embedding
     win.show()
     sys.exit(app.exec_())
 
-PyGtk embedding
+PyGObject embedding
```
Comment (McSinyx) [inline README.rst]: I believe the next line should be edited to match the length of this header.

Comment (trin94) [reply README.rst]: Thanks for pointing that out :smiley: See 69337f8

**Comment by trin94** [comment]
> Why is the Gist marked as DEPRECATED though?
> 
> Edit: sorry I missed the issue reference; when #99 is fixed, could the GTK example be updated to be, ehm, not deprecated?

I plan to add an example once #99 is fixed :smiley:

**Comment by jaseg** [comment]
Thank you for the example! :smile:

**Pull request merged**